velox/dwio/dwrf/common/Compression.h

```diff
@@ -26,8 +26,27 @@
 
 namespace facebook::velox::dwrf {
 
+using namespace dwio::common::compression;
+
 constexpr uint8_t PAGE_HEADER_SIZE = 3;
 
+FOLLY_ALWAYS_INLINE const CompressionOptions getDwrfOrcCompressionOptions(
```

Comment:
Why use FOLLY_ALWAYS_INLINE? This function is just called once when creating the compressor.

Reply:
removed